mw/notification.d.ts

```diff
@@ -22,11 +38,7 @@ declare global {
         function notify(
             message: string | JQuery | HTMLElement | HTMLElement[],
             options?: Partial<typeof notification.defaults>
-        ): {
-            pause: () => void;
-            resume: () => void;
-            close: () => void;
-        };
+        ): Promise<Notification>;
```

Comment:
It doesn't return an ES6 Promise, but a JQuery.Promise! They aren't exactly the same.